pkg/repo/cache.go

```diff
@@ -22,28 +23,46 @@ type cacheHelperOpts struct {
 	cacheDir string
 }
 
-var cacheHelperValues = cacheHelperOpts{};
+var cacheHelperValues = cacheHelperOpts{}
 
 type CacheHelper struct {
 	cacheDir string
 }
 
-func NewCacheHelper(cacheDir ...string) (*CacheHelper) {
-	if (len(cacheDir) == 0) {
-		cacheDir = append(cacheDir, cacheHelperValues.cacheDir);
-	} else if (len(cacheDir) > 1) {
+func expand(path string) (string, error) {
+	if len(path) == 0 || path[0] != '~' {
```

Comment:
I believe the recommended approach here is to:
(1) replace ~ with $HOME
(2) os.Expand(path)

This will cover more than just tilde expansion because it'll substitute all of the local environment variables.

Reply:
Good idea, I'll do that